src/lib/styles/global/utility-classes.scss

```diff
@@ -46,6 +46,13 @@
   border-radius: var(--border-radius);
 }
 
+.content-cell-disabled-island {
```

Comment:
Why "disabled"? It looks more highlighted to me.

Do we really need to do all this with the "content-cell..." utility classes?

Are we using them only for proposals?

Reply:
Yes, currently on for proposal summary. Why not, let me do it directly in nns-dapp.

Reply:
I'm ok to move it. Maybe we just need components shared between NNS and SNS proposals with the CSS inside them.

Reply:
Currently, the ProposalSummary component is used for both. I suggest thinking about it when we need to reuse these styles.